README.md

```diff
@@ -1,4 +1,77 @@
-# hubrise-webclient
-A pet JS/HTML project to see new HubRise orders in a web page
+<a href="https://www.hubrise.com/"><img src="./screenshots/hubrise-logo.png" align="left" height="48"></a>
+<br><br><br>
 
-[Demo](http://philatist-fish-74534.netlify.com)
+# HubRise {{app_name}} Template
+
+{{app_name}} is an app that prints new orders received by your HubRise location, with just a few lines of code.
+
+You can fork this repository and make changes to the code to create your own app. 
+
+
+## Try the app
+
+First, you need to [create a HubRise account](https://manager.hubrise.com/signup). It's free and it only takes a minute. 
+
+Then, simply [open the app](http://philatist-fish-74534.netlify.com) in your browser. 
+
+You will be asked to connect the app to your HubRise account. 
+Granting permissions is a necessary step in every OAuth2.0 authentication process, and requires the user to manually **Allow** the app access. 
+
+![OAuth page granting access to the app](./screenshots/oauth-page.png)
+
+You will then see a single-page app that displays the list of the latest orders received by your HubRise location. 
+Each line in the logs displays the time, id, status, and total price of the order.
+
+![Single-page app with the orders from HubRise](./screenshots/logs-page.png)
```

Comment:
Should we include instructions on how to inject orders? Maybe we could create another client, and share the id and secret in this README to save the reader from doing it.

The reader would just need to:
- Authorize the new client on his location by opening a predefined URL.
- Copy the authorization code in a predefined curl request to retrieve a token.
- Copy the token in a predefined curl request to create an order.

Reply:
I think that adding all that information in the README could steal away the attention from the purpose of this app.
I see two options here:
1. We can point the reader to the Quick start guide, where they can find pre-built Postman requests and learn how to inject orders
2. We could create a TinyOrder app, similar to this, where the user can send a predefined order to the app (maybe just changing the customer's name). So they would open the two pages, send the order from one and receive it in the other.

Reply:
I was thinking initially of redirecting the user to the Quick start, but that will lead him to take extra steps, like creating a client. We want him to see orders in TinyTablet as fast as possible.

Creating a reciprocal app to shoot orders is an interesting idea. A concern however is that it will keep the user (likely a developer) away from the API, that we ultimately want him to get interested in.

How much do you think you could shorten the steps to have another connection to shoot orders from curl?

Reply:
I've added a short paragraph under an optional section. I hope it's short enough, I think we cannot do much less than that.